.env.example

```diff
@@ -19,44 +17,31 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+# Enter a short url redirect service for invitations, it needs to redirecto to /invitation-link
+# INVITATION_SHORTLINK='example.com'
```

Comment:
+1 I would put it into the hosts section, a.k.a. # important

Reply:
Done ✅